documentation/modules/oauth/con-kafka-keycloak-authz-models.adoc

```diff
@@ -1,61 +1,29 @@
 [id="con-kafka-keycloak-authz-models_{context}"]
-= Kafka and Keycloak authorization models overview
+= Kafka and Keycloak authorization models
 
 [role="_abstract"]
-
 Kafka and Keycloak Authorization Services use different authorization models.
 
-[discrete]
-== Kafka authorization model
+* Kafka's authorization model uses _resource types_
+* The Keycloak Authorization Services model has four concepts for defining and granting permissions: 
+** _resources_ 
+** _authorization scopes_
+** _policies_
+** _permissions_
 
-Kafka's authorization model uses _resource types_. 
+.Kafka authorization model
 When a Kafka client performs an action on a broker, the broker uses the configured `KeycloakAuthorizer` to check the client's permissions, based on the action and resource type.
 
-Kafka uses five resource types to control access: `Topic`, `Group`, `Cluster`, `TransactionalId`, and `DelegationToken`.
 Each resource type has a set of available permissions.
+For example, the `Topic` resource type has `Create` and `Write` permissions among others.
 
-*Topic*
-
-* `Create`
-* `Write`
-* `Read`
-* `Delete`
-* `Describe`
-* `DescribeConfigs`
-* `Alter`
-* `AlterConfigs`
```

Comment:
Do we have these lists somewhere else? They seem to be useful.

Reply:
We had an out-of-date list. Rather than continuing to maintain it, I thought to add the following with an example and link to the full list in the Kafka docs:

> .Kafka authorization model
> When a Kafka client performs an action on a broker, the broker uses the configured `KeycloakAuthorizer` to check the client's permissions, based on the action and resource type.
> 
> Each resource type has a set of available permissions.
> For example, the `Topic` resource type has `Create` and `Write` permissions among others.
> 
> Refer to the https://kafka.apache.org/documentation/#security_authz_primitives[Kafka authorization model] in the Kafka documentation for the full list of resources and permissions.